dotcom-rendering/src/web/components/StarRatingBlockComponent.tsx

```diff
@@ -11,6 +11,7 @@ type Props = {
 const starsWrapper = css`
 	background-color: ${brandAltBackground.primary};
 	display: inline-block;
+	padding: 1px 1px 0 1px;
```

Comment:
What change does this do, the block component isn't used on fronts right?

Reply:
Good catch - Removed this